hack/docker.sh

```diff
@@ -55,6 +59,7 @@ function build_local_image() {
   echo "Building image for ${platform}: ${image_name}"
   set -x
   docker build --build-arg BINARY="${target}" \
+          --build-arg http_proxy --build-arg https_proxy --build-arg no_proxy \
```

Comment:
Did u try build without setting `http_proxy/https_proxyno_proxy`?

Reply:
It always depends on the implementation of the underlying program but usually the consensus is to not take this into account when the variable is defined but empty.

Reply:
shall we add `HTTP_PROXY`? Some programs only recognize the upper case variables.

Reply:
#2565 was preferred to this solution.
Are you ok to close this PR?

Reply:
/close